lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -39,7 +40,14 @@
     <div class="message message-status ${status_css_class} is-shown">
       <p class="message-copy">
         <%
-          certificate_available_date_string = course_overview.certificate_available_date.strftime('%Y-%m-%dT%H:%M:%S%z')
+          if settings.FEATURES.get("ENABLE_V2_CERT_DISPLAY_SETTINGS", False):
```

Comment:
These updates are great, but we need to also make sure we are sending the right updated dates to the learning MFE.

Reply:
I think this is handled already. Do you have a specific spot you're thinking about so I can verify? (Not sure if you had to deal with this for your rewording work)